src/components/AddPlaidBankAccount.js

```diff
@@ -96,12 +96,9 @@ class AddPlaidBankAccount extends React.Component {
 
     render() {
         const accounts = this.getAccounts();
-        const options = _.chain(accounts)
-            .filter(account => !account.alreadyExists)
-            .map((account, index) => ({
-                value: index, label: `${account.addressName} ${account.accountNumber}`,
-            }))
-            .value();
+        const options = _.map(accounts, (account, index) => ({
```

Comment:
Why did you remove the filter?

Reply:
The filter was just moved higher up the chain for now. 

https://github.com/Expensify/Expensify.cash/blob/125dbf3c455b37913c702bb5eb5c792bc30080ed/src/libs/actions/BankAccounts.js#L93-L94

But we are not showing any "already set up" accounts. At least for now. So, a user would need to delete that bank account in order to see those as options. There might be a better way to deal with this, but I couldn't think of one.